modxcms/revolution
# Remove 'import resources' and 'import site' from core

### What does it do?

Removes the "Import HTML" and "Import Static Resources" features from the manager, internally represented as import/index and import/resources. 

### Why is it needed?

These import features are very rarely used and [suggestions](https://github.com/modxcms/revolution/issues/13199#issuecomment-453791045) to [remove it from core](https://github.com/modxcms/revolution/issues/13737#issuecomment-358229265) and instead encourage an extra to be made has not been opposed in several years. 

I strongly agree with the sentiment expressed in those topics. This is a pretty much unused and undocumented feature with [very little improvement over the years](https://github.com/modxcms/revolution/commits/2.x/core/model/modx/import), and would be better off as an extra. 

💰 I'm happy to pitch in €100 for someone to take the code removed here, stick it in a functional extra, and release it for future users. If there's a sensible way to adapt it into ImportX I'd also be happy to review that but I'm not sure that'll translate as easily so would likely be a lot more work. 

### How to test

Build the core, run setup, confirm the rest of the manager is unaffected and the menu items have been removed. 

### Related issue(s)/PR(s)

Discussed in a few places, including #13199 and #13737. 

Fixes #13737.
